packages/datadog-plugin-cypress/test/app/cypress/plugins/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (on, config) => {
+  // We can't use the tracer available in the testing process, because this code is
+  // run in a different process. We need to init a different tracer reporting to the
+  // url set by the plugin agent
```

Comment:
it is not standard. This plugin code runs in a different process (`spawn`) and does not inherit (AFAICT) env vars from the main process. If you want variables available here you have to go through `config.env`

Reply:
> Also, why force disable startup logs and all plugins? It's possible some of them would work, especially if this code runs before the test files are loaded.

Yeah this might be interesting if users are using libraries we instrument within the plugin process.

Reply:
you're absolutely right about using the existing environment variables 👍 . As for the plugins: it might be interesting if the user is doing stuff _within_ the plugin process with libraries we instrument.